pages/tasks.js

```diff
@@ -51,7 +66,37 @@ const Tasks = ({ appContext }) => {
     ]
 
     Promise.all(invitationPromises)
-      .then((allInvitations) => setGroupedTasks(formatTasksData(allInvitations)))
+      .then(async (allInvitations) => {
+        const aERecommendationInvitations = allInvitations[2].filter((p) =>
+          p.id.endsWith('/Action_Editors/-/Recommendation')
+        )
+        const aERecommendationEdgesP = aERecommendationInvitations.map((p) =>
+          api
+            .get('/edges', {
+              invitation: `${p.domain}/Action_Editors/-/Recommendation`,
+              groupBy: 'head',
+              domain: p.domain,
```

Comment:
```suggestion
              invitation: `${p.domain}/Action_Editors/-/Recommendation`,
              groupBy: 'head',
```

Reply:
Rarely and definitely not in this case, because it doesn't help reduce the amount of documents returned.